numpy/ma/tests/test_core.py

```diff
@@ -1232,6 +1232,32 @@ def test_minmax_methods(self):
         assert_(x.max() is masked)
         assert_(x.ptp() is masked)
 
+    def test_minmax_dtypes(self):
+        # Additonal tests on max/min for non-standard float and complex dtypes
+        (x, _, a10, m1, _, xm, _, _, _, _) = self.d
+        assert_equal(xm.max(), a10)
+        assert_equal(masked_array(x, mask=m1, dtype=np.half).max(), np.half(a10))
+        assert_equal(masked_array(x, mask=m1, dtype=np.single).max(), np.single(a10))
+        assert_equal(masked_array(x, mask=m1, dtype=np.double).max(), np.double(a10))
+        assert_equal(masked_array(x, mask=m1, dtype=np.longdouble).max(), np.longdouble(a10))
+        assert_equal(masked_array(x, mask=m1, dtype=np.cfloat).max(), np.cfloat(a10))
+        assert_equal(masked_array(x, mask=m1, dtype=np.cdouble).max(), np.cdouble(a10))
+        assert_equal(masked_array(x, mask=m1, dtype=np.clongdouble).max(), np.clongdouble(a10))
+        ym = masked_array([1e20, 2e20], mask=[1,0])
+        assert_equal(ym.min(), 2e20)
```

Comment:
This test isn't interesting, it's the same as the one on line 1250

Reply:
Good point, I've removed it